components/indicatorsTable/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button } from "@/components/uikit/button"
+import { calculateBMI, deleteRecordFromCard } from "@/domains/cards/services"
+import { Card } from "@/domains/cards/types"
+type Props = {
+  id: String
```

Comment:
If you fix type `String` to `string`, you will not need `if (typeof id === "string")` anymore.